server/lib/pubsubNotifiers/notifiers/inMemory.js

```diff
@@ -0,0 +1,25 @@
+const PubSub = require('graphql-subscriptions').PubSub
+const { log } = require('../../util/logger')
+
+function InMemoryPubSub (config) {
+  const client = new PubSub()
+
+  this.publish = function publish ({ topic, compiledPayload }, context) {
+    let payload = compiledPayload(context)
+    // The InMemory pubsub implementation wants an object
```

Comment:
A comment on top of the function maybe is better in order to describe it. 

```javascript
/**
* The InMemory pubsub implementation wants an object
* etc ...
*/
```

Reply:
This particular comment isn't about the overall function, this comment is specifically about the `JSON.parse()` call underneath it.

Reply:
Then, maybe it will be more clear if be as `payload = JSON.parse(payload) // required becase of  `

If the explanation is because of some line it should be in front of it or at least on top of it. Also, IMHO usually when the code requires many comments is because the code is not clear enough which means that the vars can have better names, and/or the logic can be encapsulated, and/or maybe it is unnecessarily complex.

We could start to do things more readable and easier to keep maintained as improved. I know that it is a detail and sounds boring, however, the details matters and these details at the end make all difference to read a code and etc ..